src/qtiItem/core/Container.js

```diff
@@ -127,7 +127,10 @@ var Container = Element.extend({
                     //@todo : container rendering merging, to be tested
                     tpl = tpl.replace(elt.placeholder(), elt.render(renderer));
                 } else {
-                    tpl = tpl.replace(elt.placeholder(), '{{{' + serial + '}}}');
+                    tpl = tpl
+                        .replace(elt.placeholder(), serial)
+                        .replace('{', '&lcub;')
```

Comment:
Thought: if `'&lcub;'` is a `{` sign - can we use a standard HTML entity for this: `'&#123;' `
and possibly use the regular expression in order to match all occurrences:
`.replace(/\{/g, '&#123;')`.
Could you please test if it will work in your case?

Reply:
**thought:** I'm not sure `&#123;` is better than `&lcub;`. It is less readable. The numeric version is acceptable when replacing chars with entities using their Unicode value without bothering about a dictionary. But I'd rather suggest, whenever possible, to always use the named entities for the sake of readability.

Reply:
Good catch regarding single occurrence, but HB parse brokes only first open brace in the begin of placeholder, other `{` can be ignored.
Agreed with @jsconan `&lcub;` is more readable